django_mongodb/functions.py

```diff
@@ -57,6 +72,23 @@
 }
 
 
+def aggregate(self, compiler, connection, **extra_context):  # noqa: ARG001
+    if self.filter:
+        copy = self.copy()
+        copy.filter = None
+        source_expressions = copy.get_source_expressions()
+        condition = When(self.filter, then=source_expressions[0])
+        copy.set_source_expressions([Case(condition)] + source_expressions[1:])
+        node = copy
+    else:
+        node = self
+    lhs_mql = process_lhs(node, compiler, connection)
+    operator = MONGO_AGGREGATIONS.get(self.__class__)
+    if self.__class__ in (StdDev, Variance) and "_SAMP" in self.function:
```

Comment:
I don't think it makes a huge difference here but I believe if you use a set instead of tuple: `{StdDev, Variance}` that is slightly optimized when using `in`.